### PR TITLE
Closes #4 Add vaccines test data

### LINKED ISSUE #4: General Issue: Move any test data in {admiralvaccine} to {admiraldata}
### Background Information

Prefix: `vaccine_<dataset>`.
{admiralvaccine} contact: Neetu Sangaru @neetusan .

### Definition of Done

- SDTM contents of  `data` folder of {admiralvaccine}, as well as any relevant files in `dev` folder in copied over to same locations in {pharmaversesdtm}.
- New datasets catalogued in `data.R`.

### PR DESCRIPTION
**Pull request by ahasoplakus**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/devel/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [ ] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [ ] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [ ] Address any updates needed for vignettes and/or templates
- [ ] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [ ] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/cran-release/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: data-raw/dm_vaccine.R [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Dataset: dm_vaccine
+# Description: DM SDTM dataset for Vaccine  studies
```
Comment (manciniedoardo) [inline data-raw/dm_vaccine.R:2]: please remove extra space here

File: data-raw/ex_vaccine.R [hunk 2]
```diff
@@ -0,0 +1,67 @@
+# Dataset: ex_vaccine
+# Description: EX SDTM dataset for Vaccine  studies
```
Comment (manciniedoardo) [inline data-raw/ex_vaccine.R:2]: Remove extra space

Comment (manciniedoardo) [reply data-raw/ex_vaccine.R:2]: I think this is eleavnt for most files

**Review by manciniedoardo — CHANGES_REQUESTED** [review]
Fantastic work @ahasoplakus - as this is the first PR in what will be the new versionof pharmaversesdtm, please could you increment the version in the DESCRIPTION file to 0.2.0 and also please could you add a section in the changelog for this version and describe the test datasets that have been added?

In terms of the actual programs I only have a comment about a spurious space in the titles of the programs but happy for you to ignore as it's super minor.

Please note that the next, equally important, step once this PR is merged is removing the test data from admiralvaccine and making sure that admiralvaccine references pharmaversesdtm instead. This will need to be done before the next admiralvaccine release (I assume december?)

File: data-raw/ex_vaccine.R (hunk 2 above)
Comment (ahasoplakus) [reply data-raw/ex_vaccine.R]: Updated for all. Thanks!

**Comment by ahasoplakus** [comment]
> Fantastic work @ahasoplakus - as this is the first PR in what will be the new versionof pharmaversesdtm, please could you increment the version in the DESCRIPTION file to 0.2.0 and also please could you add a section in the changelog for this version and describe the test datasets that have been added?
> 
> In terms of the actual programs I only have a comment about a spurious space in the titles of the programs but happy for you to ignore as it's super minor.
> 
> Please note that the next, equally important, step once this PR is merged is removing the test data from admiralvaccine and making sure that admiralvaccine references pharmaversesdtm instead. This will need to be done before the next admiralvaccine release (I assume december?)

Hi @manciniedoardo I have bumped the package version and updated the changelog, thanks a lot for your review. Will implement the changes required in {admiralvaccine} once this PR is merged

**Comment by bundfussr** [comment]
> as this is the first PR in what will be the new versionof pharmaversesdtm, please could you increment the version in the DESCRIPTION file to 0.2.0

@manciniedoardo , shouldn't the version be set to 0.1.1.9000? Or do we use a different branching/version strategy than in admiral?

**Comment by manciniedoardo** [comment]
> > as this is the first PR in what will be the new versionof pharmaversesdtm, please could you increment the version in the DESCRIPTION file to 0.2.0
> 
> @manciniedoardo , shouldn't the version be set to 0.1.1.9000? Or do we use a different branching/version strategy than in admiral?

We are using the same branching strategy as admiral, but this isn't a patch it's a new version of the package hence the 0.2.0 version.

**Comment by bundfussr** [comment]
> > > as this is the first PR in what will be the new versionof pharmaversesdtm, please could you increment the version in the DESCRIPTION file to 0.2.0
> > 
> > 
> > @manciniedoardo , shouldn't the version be set to 0.1.1.9000? Or do we use a different branching/version strategy than in admiral?
> 
> We are using the same branching strategy as admiral, but this isn't a patch it's a new version of the package hence the 0.2.0 version.

Yes, but as we are merging directly to `main`, we should use 0.1.1.9000 to indicate that it is the development version and not a released version. (At least in admiral we are doing it this way.)

**Comment by manciniedoardo** [comment]
> > > > as this is the first PR in what will be the new versionof pharmaversesdtm, please could you increment the version in the DESCRIPTION file to 0.2.0
> > > 
> > > 
> > > @manciniedoardo , shouldn't the version be set to 0.1.1.9000? Or do we use a different branching/version strategy than in admiral?
> > 
> > 
> > We are using the same branching strategy as admiral, but this isn't a patch it's a new version of the package hence the 0.2.0 version.
> 
> Yes, but as we are merging directly to `main`, we should use 0.1.1.9000 to indicate that it is the development version and not a released version. (At least in admiral we are doing it this way.)

Thanks for the correction @bundfussr I understand now - @ahasoplakus feel free to modify and then I will approve the PR.

**Comment by ahasoplakus** [comment]
> > > > > as this is the first PR in what will be the new versionof pharmaversesdtm, please could you increment the version in the DESCRIPTION file to 0.2.0
> > > > 
> > > > 
> > > > @manciniedoardo , shouldn't the version be set to 0.1.1.9000? Or do we use a different branching/version strategy than in admiral?
> > > 
> > > 
> > > We are using the same branching strategy as admiral, but this isn't a patch it's a new version of the package hence the 0.2.0 version.
> > 
> > 
> > Yes, but as we are merging directly to `main`, we should use 0.1.1.9000 to indicate that it is the development version and not a released version. (At least in admiral we are doing it this way.)
> 
> Thanks for the correction @bundfussr I understand now - @ahasoplakus feel free to modify and then I will approve the PR.



> > > > > as this is the first PR in what will be the new versionof pharmaversesdtm, please could you increment the version in the DESCRIPTION file to 0.2.0
> > > > 
> > > > 
> > > > @manciniedoardo , shouldn't the version be set to 0.1.1.9000? Or do we use a different branching/version strategy than in admiral?
> > > 
> > > 
> > > We are using the same branching strategy as admiral, but this isn't a patch it's a new version of the package hence the 0.2.0 version.
> > 
> > 
> > Yes, but as we are merging directly to `main`, we should use 0.1.1.9000 to indicate that it is the development version and not a released version. (At least in admiral we are doing it this way.)
> 
> Thanks for the correction @bundfussr I understand now - @ahasoplakus feel free to modify and then I will approve the PR.



> > > > > as this is the first PR in what will be the new versionof pharmaversesdtm, please could you increment the version in the DESCRIPTION file to 0.2.0
> > > > 
> > > > 
> > > > @manciniedoardo , shouldn't the version be set to 0.1.1.9000? Or do we use a different branching/version strategy than in admiral?
> > > 
> > > 
> > > We are using the same branching strategy as admiral, but this isn't a patch it's a new version of the package hence the 0.2.0 version.
> > 
> > 
> > Yes, but as we are merging directly to `main`, we should use 0.1.1.9000 to indicate that it is the development version and not a released version. (At least in admiral we are doing it this way.)
> 
> Thanks for the correction @bundfussr I understand now - @ahasoplakus feel free to modify and then I will approve the PR.

@manciniedoardo Set the package version to 0.1.1.9000 as discussed, however the CMD checks have failed with a NOTE 
image

**Comment by manciniedoardo** [comment]
> > > > > > as this is the first PR in what will be the new versionof pharmaversesdtm, please could you increment the version in the DESCRIPTION file to 0.2.0
> > > > > 
> > > > > 
> > > > > @manciniedoardo , shouldn't the version be set to 0.1.1.9000? Or do we use a different branching/version strategy than in admiral?
> > > > 
> > > > 
> > > > We are using the same branching strategy as admiral, but this isn't a patch it's a new version of the package hence the 0.2.0 version.
> > > 
> > > 
> > > Yes, but as we are merging directly to `main`, we should use 0.1.1.9000 to indicate that it is the development version and not a released version. (At least in admiral we are doing it this way.)
> > 
> > 
> > Thanks for the correction @bundfussr I understand now - @ahasoplakus feel free to modify and then I will approve the PR.
> 
> > > > > > as this is the first PR in what will be the new versionof pharmaversesdtm, please could you increment the version in the DESCRIPTION file to 0.2.0
> > > > > 
> > > > > 
> > > > > @manciniedoardo , shouldn't the version be set to 0.1.1.9000? Or do we use a different branching/version strategy than in admiral?
> > > > 
> > > > 
> > > > We are using the same branching strategy as admiral, but this isn't a patch it's a new version of the package hence the 0.2.0 version.
> > > 
> > > 
> > > Yes, but as we are merging directly to `main`, we should use 0.1.1.9000 to indicate that it is the development version and not a released version. (At least in admiral we are doing it this way.)
> > 
> > 
> > Thanks for the correction @bundfussr I understand now - @ahasoplakus feel free to modify and then I will approve the PR.
> 
> > > > > > as this is the first PR in what will be the new versionof pharmaversesdtm, please could you increment the version in the DESCRIPTION file to 0.2.0
> > > > > 
> > > > > 
> > > > > @manciniedoardo , shouldn't the version be set to 0.1.1.9000? Or do we use a different branching/version strategy than in admiral?
> > > > 
> > > > 
> > > > We are using the same branching strategy as admiral, but this isn't a patch it's a new version of the package hence the 0.2.0 version.
> > > 
> > > 
> > > Yes, but as we are merging directly to `main`, we should use 0.1.1.9000 to indicate that it is the development version and not a released version. (At least in admiral we are doing it this way.)
> > 
> > 
> > Thanks for the correction @bundfussr I understand now - @ahasoplakus feel free to modify and then I will approve the PR.
> 
> @manciniedoardo Set the package version to 0.1.1.9000 as discussed, however the CMD checks have failed with a NOTE image

@bundfussr I believe we can ignore this but do you know why it's happening?

**Comment by bundfussr** [comment]
> @bundfussr I believe we can ignore this but do you know why it's happening?

Yes, it is due to our new branching strategy. We do not have a good solution yet. As a work-around we are accepting notes in the R-CMD checks at the moment. I have update the workflow.

**Review by manciniedoardo — APPROVED** [review]
